drivers/display/display_otm8009a.c

```diff
@@ -404,7 +403,7 @@ static int otm8009a_configure(const struct device *dev)
 	/* Pump 1 min and max DM */
 	buf[0] = 0x08;
 	buf[1] = 0x66;
-	buf[2] = 0x83;
+	buf[2] = 0x80;
```

Comment:
Impact on comment ?

Reply:
`0x80` is the default value for 131st parameter. If add `0x03`, it will enable `en_C24` and `en_C32` bit. The original comment not montioned it.

![Screenshot from 2023-12-19 21-09-13](https://github.com/zephyrproject-rtos/zephyr/assets/40390867/36ce9e4e-d2ad-4d7e-ab98-cac268a823d9)

Reply:
My request was to update the comment as it will now be wrong.

Reply:
no impact on the comment